hellosign_sdk/hsclient.py

```diff
@@ -1885,6 +1886,8 @@ def _send_signature_request(self, test_mode=False, client_id=None, files=None,
                 order (str, optional): The order the signer is required to sign in
                 pin (str, optional): The 4- to 12-character access code that will secure this signer's signature page
 
+            custom_fields (list of dict, optional): A list of custom fields. Required when a CustomField exists in the Template. An item of the list should look like this: `{'name: value'}`
```

Comment:
I don't think this example is really adding anything. It's just a dict with `name` as the key, and `value` as the value. 

```suggestion
            custom_fields (list of dict, optional): A list of custom fields. Required when a CustomField exists in the Template
```